README.md

```diff
@@ -613,51 +615,97 @@ There are many options for training the model with your synthetic data-enhanced
+3. `accelerated` uses the instructlab-training library which supports GPU accelerated and distributed training. The `full` loop and data processing functions are either pulled directly from or based off of the work in this library.
+
+#### Train the model locally
 
 ```shell
 ilab model train
 ```
 
 > **NOTE:** ⏳ This step can potentially take **several hours** to complete depending on your computing resources. Please stop `ilab model chat` and `ilab model serve` first to free resources.
 
-If you are using `ilab model train --legacy` or are on MacOS:
+If you are using `ilab model train --pipeline=simple` on Linux:
 
 `ilab model train` outputs a brand-new model that can be served in the `models` directory called `ggml-model-f16.gguf`.
 
-If you are using `ilab model train` with a GPU enabled system:
+If you are using `ilab model train --pipeline=simple` on MacOS:
+
+`ilab model train` outputs a brand-new model that is saved in the `<model_name>-mlx-q` directory.
+
+If you are using `ilab model train --pipeline=full` on Linux or MacOS:
+
+`ilab model train` outputs brand-new `.bin` and `.gguf` models in the `~/.local/share/instructlab/checkpoints/hf_format` directory. in each `samples_*` directory you will find two GGUF models. One is quantized and one is full precision, the quantized one has a Q4-M-K suffix.
+
+If you are using `ilab model train --pipeline=accelerated` on Linux:
 
 `ilab model train` outputs brand-new models that can be served in the `~/.local/share/instructlab/checkpoints` directory.  These models can be run through `ilab model evaluate` to choose the best one.
 
-If you are using `ilab model train --strategy lab-multiphase`
+If you are using `ilab model train --strategy lab-multiphase` on Linux:
 
-#### Train the model locally on an M-series Mac
+`ilab model train` outputs brand-new models that can be served in `~/.local/share/instructlab/phased/phase1/checkpoints` and `~/.local/share/instructlab/phased/phase2/checkpoints`. Phase 1 correlates to knowledge training and phase 2 correlates to skills training.
 
-To train the model locally on your M-Series Mac is as easy as running:
+When running multi phase training evaluation is run on each phase, we will tell you which checkpoint in this folder performs the best.
+
+#### Train the model locally on an M-series Mac or on Linux using the full pipeline
+
+To train the model locally on your M-Series Mac using our full pipeline and MPS or on your Linux laptop/deskop using CPU:
 
 ```shell
-ilab model train
+ilab model train --pipeline full --device mps
+```
+
+```shell
+ilab model train --pipeline full --device cpu
+```
+
+> **Note:** ⏳ This process will take a while to complete. If you run for ~8 epochs it will take approximately 5-8 hours on a 64 GB machine.
```

Comment:
Is 5-8 hours on a 64GB machine the right metric?  Seems like compute power should factor in as well.

Reply:
I would avoid stating any number of hours. There are way too many variables: additional workloads, CUDA versions, compute capability. It's better to just say `several hours`. That way you don't disappoint anyone.

Reply:
I removed specific amounts and switched to `serveral hours`